examples/resque_scheduler/cookbooks/custom-redis/README.md

```diff
@@ -0,0 +1,3 @@
+# Custom Redis
+
+This custom-redis recipe is used in the resque example to install redis on the utility instance named "resque" instead of the default "redis".
```

Comment:
Shall we include this small Redis wrapper at all? Leaning towards adding a section on the Resque-Scheduler wrapper's Readme telling the customer to enable the full Redis recipe (as we do with Sidekiq).